IfaceMgr/IfaceMgr.cpp

```diff
@@ -487,6 +487,40 @@ void TIfaceMgr::notifyScripts(const std::string& scriptName, SPtr<TMsg> question
     notifyScript(scriptName, action, params);
 }
 
+/* Notify script invoked when expiry occurs and lifetimes are reset.
+   This method can be modified as per the client requirements.
+*/
+void TIfaceMgr::notifyScripts(const std::string& scriptName, TNotifyScriptParams& params, TIAType type, char * addr, int len)
+{
+    if (!scriptName.length()) {
+        Log(Debug) << "Not executing external script (Notify script disabled)." << LogEnd;
+        return;
+    }
+
+    string action = "add";
+
+    switch (type) {
+        case IATYPE_IA :
+            Log(Info) << "Notify script invoked based on expired values for IA_NA." << LogEnd;
+            params.addParam("ADDR1", addr);
```

Comment:
You always assume there's one address. What if there are multiple addresses? The next one should be ADDR2, possibly ADDR3 etc.

Reply:
I have corrected it.